src/sound/manager.js

```diff
@@ -43,6 +43,17 @@ pc.extend(pc, function () {
 
             if (this.context) {
                 var context = this.context;
+
+                // resume AudioContext on user interaction because of new Chrome autoplay policy
+                var resumeContext = function () {
+                    context.resume();
+                    window.removeEventListener('mousedown', resumeContext);
+                    window.removeEventListener('touchend', resumeContext);
+                };
+
+                window.addEventListener('mousedown', resumeContext);
+                window.addEventListener('touchend', resumeContext);

```

Comment:
Google seem to be just recommending use of the 'click' handler: https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#webaudio  ...one instead of two events.

Reply:
Hm I'm not sure if the click event is fired on a document when you just have touch input I need to check...

Reply:
I'm fairly sure it is, but yeah, do check.

Reply:
If touch event is handled or `preventsDefault` is called, then I believe `click` event might not fire.
There is another situation where `window` will not get events, if developer of the app have handled `touchend` or `mousedown` events and called `stopPropagation` on them. Which will lead to `window` never getting bubbled event.

Reply:
Not sure there is a way to list all current events and temporary cancel them before first event intercepted. But that is magical and intrusive as well.

Looks like best practice would be to offer an exposed method for a developer to call himself as well. And document browsers behaviour and how to enable sound, so developer will have to adopt those practices them self as well.

One of easiest way to implement by developer would be "press start" - title screen, that is common practice since old school 8-bit games :)